app/views/capture/index2.html.erb

```diff
@@ -224,6 +224,7 @@
                 <p style="margin-left: 1.8em;margin-top:0rem;width: 96%;">This display shows the last few seconds of data, descending. The top, newest row of pixels is used by default to generate the graph. This can be changed later.</p>
               </div>
               
+              <input name="dataurl" type="hidden" id="dataurl" />
```

Comment:
This caused the test failure by creating a second instance of the dataurl input. Removing as it's already present in the save form!

Reply:
```suggestion
```